plugins/xp-grapher

```diff
@@ -1,2 +1,2 @@
 repository=https://github.com/cosmicquahog/XpGrapher.git
-commit=8ba7fb9f65db32353ffcc794f4626a796900c889
+commit=128f740ec19ef0ec4aa0e7847e69b13a1533cd90
```

Comment:
This will keep the bot from labeling your PRs as non-author, I see from the commits that you have write permissions, as mentioned in that screenshot.
```suggestion
commit=128f740ec19ef0ec4aa0e7847e69b13a1533cd90
authors=cosmicquahog,Raymoz101
```